caluma/form/schema.py

```diff
@@ -62,11 +66,29 @@ class Meta:
 
 
 class IntegerQuestion(graphene.ObjectType):
+    max_value = graphene.Int()
+    min_value = graphene.Int()
+
+    def resolve_max_value(self, info):
+        return self.configuration["max_value"]
+
+    def resolve_min_value(self, info):
+        return self.configuration["min_value"]
```

Comment:
You don't need those - delete 'em!

Reply:
True. Deleted. Also changed that max and min values are non null in query as we always have a default.